src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -384,8 +376,6 @@ def preprocess(
         image_mean: Optional[Union[float, List[float]]] = None,
         image_std: Optional[Union[float, List[float]]] = None,
         do_pad: Optional[bool] = None,
-        do_center_crop: Optional[bool] = None,
-        crop_size: Dict[str, int] = None,
```

Comment:
We can't just remove these like this as this will break things for users. We need to still accept them and emit a warning if they are set